kpi/tests/api/v2/test_api_collections.py

```diff
@@ -307,6 +307,48 @@ def test_collection_subscribe(self):
             response.data['results'][0]['url'].endswith(pub_coll_url)
         )
 
+    def test_get_subscribed_collection(self):
+        public_collection = Asset.objects.create(
+            asset_type=ASSET_TYPE_COLLECTION,
+            name='public collection',
+            owner=self.someuser,
+        )
+        public_collection_asset = Asset.objects.create(
+            asset_type=ASSET_TYPE_TEMPLATE,
+            name='public asset',
+            owner=self.someuser,
+            parent_id=public_collection.pk
```

Comment:
could be `parent=public_collection` for the same effect :)

Reply:
changed it in the above test as well :P